openj9-11-rhel8.yaml

```diff
@@ -51,10 +51,10 @@ modules:
     version: "7"
   - name: jboss.container.jolokia
     version: "7"
-  - name: jboss.container.java.s2i.bash
+  - name: jboss.container.dnf
   - name: jboss.container.maven
-    version: "8.0.3.5"
-  - name: jboss.container.java.singleton-jdk
```

Comment:
I haven't tried a build yet to test, but you might still want this module (at the end of the list) as I think maven will still pull in openjdk.